internal/testing/e2e/cluster.go

```diff
@@ -170,35 +166,20 @@ func fileExists(name string) bool {
 	return err == nil
 }
 
-func newClusterOM(kubeconfig string, kubeconfigFromEnv string, namespace string) (*clusterOM, error) {
-	if !fileExists(kubeconfig) && fileExists(kubeconfigFromEnv) {
-		kubeconfig = kubeconfigFromEnv
-	}
-	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+func createZygote(m *testing.M) (OM, error) {
+	// creates the in-cluster config
+	kubeconfig, err := rest.InClusterConfig()
```

Comment:
For my understanding - can we simplify this to in cluster config because now we are already in context of the created CI cluster?

Reply:
I am using the in cluster config by calling `rest.InClusterConfig()`.